### ISSUE TITLE
text formatting under terms of use needs fixing

### ISSUE THREAD

**Issue by elliec** [geotagx/geotagx-theme#12]
The text formatting under terms of use (http://spmohanty-geotagx-1.cern.ch/help/terms-of-use) needs fixing, the lines should go all the way across the page, right now they look like they are squashed into a column but sometimes there is one line that goes all the way across.

I think this is the same for the API, privacy, licenses

**Comment by spMohanty** [comment]
cb15f72ea431e6fbb86256be02566e442b6320df also fixes the layout issues to some extend. Though it will need some manual intervention to probably add relevant images, and style content to make it more readable.
@supranove can you have a look at this when you are free ?

**Comment by othieno** [comment]
@spMohanty sure thing.

**Comment by elliec** [comment]
@spMohanty @supranove I think this is fixed?

**Comment by spMohanty** [comment]
@elliec : Yeah the layout issue is fixed. But I would close this issue only after wrapping up the theming of the content on the individual pages.  It seems like a lot of text and is not really readable !! Though not a high priority task for now.

**Comment by elliec** [comment]
@spMohanty ok agreed, really low priority though, terms of use pages are usually pretty text heavy so i dont think its a huge problem

**Comment by spMohanty** [comment]
Closing this issue as suggested by @elliec .
